tableauserverclient/server/endpoint/server_info_endpoint.py

```diff
@@ -0,0 +1,21 @@
+from .endpoint import Endpoint
+from ...models import ServerInfoItem
+import logging
+
+logger = logging.getLogger('tableau.endpoint.server_info')
+
+
+class ServerInfo(Endpoint):
+    def __init__(self, parent_srv):
+        super(Endpoint, self).__init__()
```

Comment:
We need to go through and update all of these. This is not the correct way to do it. "Endpoint" should be "ServerInfo". It doesn't matter since Endpoint and object both have empty constrctors. Which leaves us with ... we should just delete all of these.